server/src/test/java/org/elasticsearch/index/mapper/vectors/DenseVectorFieldMapperTests.java

```diff
@@ -77,8 +77,9 @@ protected void minimalMapping(XContentBuilder b) throws IOException {
         if (elementType != ElementType.FLOAT) {
             b.field("element_type", elementType.toString());
         }
+        b.field("index", indexed);
```

Comment:
This isn't the minimal mapping - we could do without the `index` field. However, this would make `testDeprecatedBoost` fail, as it uses `minimalMapping` to create a mapping where index defaults to false.

We could create an additional method to generate the mapping for `testDeprecatedBoost`, but seemed overkill. The actual minimal params are tested in this class.

Reply:
There is no version information in this method 😞 . We can create new methods used in the superclass in order to create different mappings for this, but I wanted to have some reviews and consider alternatives first.

Reply:
I added [a PR](https://github.com/carlosdelest/elasticsearch/pull/6) to this one with the differences for including a different method in `testDeprecatedBoost()` that would get rid of this - LMKWYT!

Reply:
I referenced in that PR, but it seems to me we need a new `minimalMapping` method that accepts a `IndexVersion` parameter. It can default to ignoring that parameter and just call the existing `minimalMapping` method. 

Then tests in `MapperTestCase` that need minimal mappings referring to a version (like deprecated boosts), they can pass them.

Reply:
That's another option I checked, and it makes more sense 👍 

If we make the `index` parameter explicit though, it is not necessary. As we both think it's better to have `index` explicit, I think we can keep the code simpler this way and not introduce additional methods in `MapperTestCase`.

Happy to revisit if you think it's worth including 👍